lib/matplotlib/image.py

```diff
@@ -822,6 +822,27 @@ def get_filterrad(self):
 
 
 class AxesImage(_ImageBase):
+    """
+    Parameters
+    ----------
+    cmap : colors.Colormap
+    norm : colors.Normalize
+        Maps luminance to 0-1
+    origin :
+    extent : tuple
+        The data axes (left, right, bottom, top) for making image plots
+        registered with data plots.  Default is to label the pixel
+        centers with the zero-based row and column indices.
+    filternorm :
+    filterrad :
+    resample : bool
+    **kwargs :
```

Comment:
```suggestion
    **kwargs
```
If you don't have a type, you can omit the colon.